wire-ios-data-model/Support/Sources/ModelHelper.swift

```diff
@@ -55,24 +55,24 @@ public struct ModelHelper {
 
     @discardableResult
     public func createSelfClient(
-        id: UUID = .init(),
+        id: String = .randomAlphanumerical(length: 8),
```

Comment:
Just this one. Why change from UUID to generating random string? A brief explanation could help here

Reply:
fyi: I think next to `randomAlphanumerical(length:)` I have added convenience methods `randomClientIdentifier()` to avoid the "magic" number in there 👍